### PR TITLE
add alt nav, PDF for LSM, adjust total numbers

### PR DESCRIPTION
**Pull request by gtarney**

### CODE REVIEW

File: app/firstlook/leaderboard/template.hbs [hunk 1]
```diff
@@ -5,11 +5,23 @@
...
+      </ul>
+    </div>
+    </section>
+
 
     <section id="comapnyGoal" class="" style="">
       <div  class=" container"  >
       <div class="row col-xs-12" style="text-align:right; display:block;">
-         <div style=" " class="lastUpdated"><p class=" dkBlue"><span style="color: white;">Last Update:</span> Thurs Jan 25, 4:00pm cst</p></div>
+         <div style=" " class="lastUpdated"><p class=" dkBlue"><span style="color: white;">Last Update:</span> Thurs. Jan 31, 400pm cst</p></div>
```
Comment (ShaunWMusic) [inline app/firstlook/leaderboard/template.hbs]: can you update this to "4:00pm"?

File: app/firstlook/leaderboard/template.hbs [hunk 2]
```diff
@@ -95,6 +107,7 @@
      <div class="col-lg-12" style="text-align: left;"><h1>LEADERBOARD<span class="dkBlue">/LSM</span></h1>
        <p class="footnote" style="padding-left: 5px;">1st Place in each Region based on % to goal</p>
      </div>
+     <div id="pdf"  class="col-sm-3 col-lg-12 "> <a href="/assets/img/Firstlook_OTBNetRev_byAE_wk4_012518_v3-4.pdf" target="_blank" style="color:#0099D8;"><img src="/assets/img/pdf-download.png" alt="download pdf"/></a></div>
```
Comment (ShaunWMusic) [inline app/firstlook/leaderboard/template.hbs]: Their should be a section in standings with pdf already in there.

File: app/firstlook/leaderboard/template.hbs [hunk 3]
```diff
@@ -23,13 +35,13 @@
...
+    	 	 <div style="float:left;"> <h2 style="color:#003057;">COMPANY-WIDE PROGRESS</h2></div>   <div style="float:right; color:#fff;"><h2 style="color:#fff;">70.97% </h2></div>
           <div class="progress" style="clear:both; background-color:white; border:1px solid #003057;">
-            <div class="progress-bar" role="progressbar" aria-valuenow="62.93" aria-valuemin="0" aria-valuemax="100" style=" width:62.93%; ">
+            <div class="progress-bar" role="progressbar" aria-valuenow="70.97" aria-valuemin="0" aria-valuemax="100" style=" width:70.97%; ">
               <!-- <span class="">70% </span> -->
             </div>
           </div>
-          <div style="float:left; margin-top:10px"><h2 style="color:#fff;">2018 GOAL</h2></div>   <div style="float:right;margin-top:10px"><h2 style="color:#003057;">$320,971,558 / $510M</h2></div>
+          <div style="float:left; margin-top:10px"><h2 style="color:#fff;">2018 GOAL</h2></div>   <div style="float:right;margin-top:10px"><h2 style="color:#003057;">$361,992,371 / $510M</h2></div>
```
Comment (ShaunWMusic) [inline app/firstlook/leaderboard/template.hbs]: I gave you the wrong numbers for this. should be $365,844,137

File: app/firstlook/leaderboard/template.hbs [hunk 4]
```diff
@@ -23,13 +35,13 @@
       </div>
     </div>
     	<div class="col-xs-12 col-md-8 progressBox" >
-    	 	 <div style="float:left;"> <h2 style="color:#003057;">COMPANY-WIDE PROGRESS</h2></div>   <div style="float:right; color:#fff;"><h2 style="color:#fff;">62.93% </h2></div>
+    	 	 <div style="float:left;"> <h2 style="color:#003057;">COMPANY-WIDE PROGRESS</h2></div>   <div style="float:right; color:#fff;"><h2 style="color:#fff;">70.97% </h2></div>
```
Comment (ShaunWMusic) [inline app/firstlook/leaderboard/template.hbs]: this percentage should be changed to 74.58%

File: app/firstlook/leaderboard/template.hbs [hunk 5]
```diff
@@ -23,13 +35,13 @@
...
           <div class="progress" style="clear:both; background-color:white; border:1px solid #003057;">
-            <div class="progress-bar" role="progressbar" aria-valuenow="62.93" aria-valuemin="0" aria-valuemax="100" style=" width:62.93%; ">
+            <div class="progress-bar" role="progressbar" aria-valuenow="70.97" aria-valuemin="0" aria-valuemax="100" style=" width:70.97%; ">
```
Comment (ShaunWMusic) [inline app/firstlook/leaderboard/template.hbs]: same here: 74.58%

**Comment by gtarney** [comment]
updated

**Pull request merged**